plugins/aws-appmesh/plugin/commands.go

```diff
@@ -173,11 +173,12 @@ func (plugin *Plugin) setupEgressRules(
 	}
 
 	if config.EgressIgnoredPorts != "" {
-		err = iptable.Append("nat", egressChain, "-p", "tcp", "-m", "multiport", "--dports",
-			config.EgressIgnoredPorts, "-j", "RETURN")
-		if err != nil {
-			log.Errorf("Append rule for egressIgnoredPorts failed: %v", err)
-			return err
+		for _, port := range config.EgressIgnoredPorts {
```

Comment:
Are there fields other than EgressIgnoredPorts that have a similar "15-entires" limitation? It seems like AppPorts has the same problem since it uses the mport extension.

Reply:
yeah it looks like that would be limited to 15 ports as well, although i dont know if that limitation is actually affecting anybody. id be happy to reuse the chunking there as well if youre onboard

Reply:
Yes, please. It is much cheaper to fix all similar bugs now.

Reply:
ok sounds good. Ive updated the PR to use a local function thats reused in both places. I toyed around with an alternative method like creating a new append method on the iptable struct to handle multiport appends and chunk within that method, but going down that route it was messy bc we would need to parse the variatic rulespec parameter... but let me know what you think @ofiliz